src/servers/grpc_server.cc

```diff
@@ -292,6 +292,59 @@ GRPCServer::Create(
   return Status::Success;
 }
 
+Status
+GRPCServer::CreateUniqueEndpointPorts(
+    InferenceServer* server, std::string endpoint_name, uint16_t port,
+    std::unique_ptr<GRPCServer>* grpc_server)
+{
+  // DLIS-162 - provide global defaults and cli overridable options
+  g_Resources = std::make_shared<AsyncResources>(
+      server,  // InferenceServer*,
```

Comment:
We don't use this commenting style. Put a comment before the statement if necessary. For constants like the 1's use an inline comment, ... 1 /* infer threads */, ...